heroku/heroku-vscode
# Jw/heroku auth

This PR provides auth for Heroku

## Testing
1. Make sure you have the Heroku CLI installed on your machine.
2. Make sure you are signed out using `heroku auth:logout`.
3. Checkout this branch, install deps, and run `npm run watch`
4. Launch the extension.
5. On the far left toolbar, click on the Heroku Icon. You should see a button to sign into your Heroku Account. Click the Sign in button
<img width="332" alt="image" src="https://github.com/user-attachments/assets/e81b7e38-5383-45e5-9aed-27972b4ad0ad">

6.  Follow the auth process in the browser
7. Once you have successfully authenticated, you should see the Sign in button removed and your Heroku account shown when clicking on the Accounts icon 
<img width="267" alt="image" src="https://github.com/user-attachments/assets/53872c1d-7900-4072-95a6-707bcfb1ad37">
